content/basics/ably.textile

```diff
@@ -0,0 +1,100 @@
+
+Read more in our "Pub/Sub section":/channels/.
+
+h3(#presence-and-occupancy). Presence and Occupancy
+
+Ably’s presence feature allows clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events (such as entering, updating their state, or leaving the channel) in real time using our realtime SDKs, or via the Ably Integrations. You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the REST API.
+
+Occupancy provides metrics about the clients attached to a channel, including a count of the number of clients attached, the number of publishers and subscribers, the number of presence members, and the number of presence publishers and subscribers.
+
+Presence provides information about the activity of members entered into the presence set of a channel. Presence members announce their state to presence subscribers via presence events. For example, when entering the presence set, leaving the presence set, or updating their member data.
+
+Read more in our "Presence and occupancy section":/presence-occupancy/.
+
+h3(#message-storage-and-history). Message storage and History
+
+By default messages are only stored for two minutes on Ably’s servers, these messages can be retrieved in the event of a lost connection using the History API. There are several options available to increase this storage time. One option is to enable persistence of all messages, for typically 24 hours. These messages can also be retrieved using the History API. Persistence of the last message is also available, although this is retrieved using the rewind channel option. Finally, it is possible to persist messages to external storage services, using the various integrations that are available.
+
+Read more in our "Message storage and history section":/storage-history/.
+
+h3(#push-notifications). Push Notifications
+
+Ably can deliver push notifications to devices using, amongst others, Apple’s Push Notification service and Google’s Firebase Cloud Messaging service. Push notifications, unlike Ably’s channel based pub/sub messaging, do not require the device to maintain a connection to Ably, as the underlying platform or OS is responsible for maintaining its own battery-efficient transport to receive push notifications. Therefore, push notifications are commonly used to display visual notifications to users or launch a background process for an app in a battery-efficient manner.
+
+Read more in our "Push notifications section":/push/.
+
+h2(#infrastructure). About the Infrastructure
+
+The importance of a seamless experience can't be overstated, that's why we've built the Ably infrastructure around the "Four Pillars of Dependability":https://ably.com/four-pillars-of-dependability :
+
+* "Performance":#performance
+* "Integrity":#integrity
+* "Reliability":#reliability
+* "Availability":#availability
+
+h3(#performance). Performance
```

Comment:
These 4 sections can be heavily influence by the content of the four pillars, but I'm not sure they should just be a copy and paste of the existing content. The tone of voice is noticeably different.